content/en/docs/developer-guide/grpc/_index.md

```diff
@@ -2,6 +2,8 @@
 title: gRPC API
 linktitle: gRPC API
 description: Enable and configure the gRPC capabilities of Falco
+aliases:
+- ../../grpc
```

Comment:
This redirects to https://deploy-preview-1362--falcosecurity.netlify.app/grpc/ instead of https://deploy-preview-1362--falcosecurity.netlify.app/docs/grpc/ (see: https://falco.org/docs/grpc/ )